### PR TITLE
NavBarCreated

### PR DESCRIPTION
**Pull request by Lakiajo**
NavBar created under navbar branch. Placeholder image added. Search Bar added, Clear messages button created. Sticky-top class added.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,12 @@
   <title>Baratheon</title>
 </head>
 <body>
-  <button class="btn-danger">hey</button>
+  <nav class="navbar sticky-top navbar-light bg-light">
+    <form class="d-flex flex-row justify-content-between col-12">
+      <img src="https://f4.bcbits.com/img/0009711024_10.jpg" width="50" height="50" class="flex-row align-top col-1" id="navImg">
+      <input class="form-control mr-sm-2 col-5" id="chatInput" type="search" placeholder="Search" aria-label="Search">
+      <button class="btn btn-outline-success my-2 my-sm-0 col-1" id="clearChat" type="submit">Clear Chat</button>
```
Comment (HeathJHMoore) [inline src/index.html]: we probably will want the take the `type='submit'` off of this button since it causes the page to refresh when you click it

**Review by HeathJHMoore — CHANGES_REQUESTED** [review]

**Review by Lakiajo — COMMENTED** [review]
Looks good but we will need to figure out why changing size of the webpage decreases the image size. Likely can be added with scss.

File: src/index.html (hunk 1 above)
Comment (Lakiajo) [reply src/index.html]: This has been updated.

**Comment by Lakiajo** [comment]
closes #22

**Pull request merged**